.github/workflows/simulation_test.yml

```diff
@@ -201,6 +233,64 @@ jobs:
         mpirun -np 2 special -mpi -python $NEURODAMUS_PYTHON/init.py --configFile=simulation_sonata_coreneuron.json
         ls reporting_coreneuron/*.h5
 
+    - name: Install test and doc dependencies
+      run: |
+        . ./venv/bin/activate
+        pip install -r tests/requirements.txt
+        pip install "sphinx<5.1.0" sphinx-bluebrain-theme
+
+    - name: test-unit
+      run: |
+        . ./venv/bin/activate
+        if [[ ${{ matrix.python-version }} == 3.9.* ]]; then
+          export RDMAV_FORK_SAFE=1
+        fi;
```

Comment:
This is heavily repeated. Can we automatically set that environment variable? And is the virtual env not active from the step before?